tests/e2e/dsc_deletion_test.go

```diff
@@ -88,50 +88,18 @@ func (tc *testContext) testApplicationDeletion(component components.ComponentInt
 func (tc *testContext) testAllApplicationDeletion() error {
```

Comment:
i start to think, why we did not use parallelism  when we test the deletion?

Reply:
> i start to think, why we did not use parallelism when we test the deletion?

Good question. I have no idea. In practice if in the PollUntilContextTimeout() call replace immediate to "true" it will take basically time of the slowest one + n * List request time, not a big difference. But I can make a follow up and organise them as separate parallel tests like in creation.